fn_ioc_parser/setup.py

```diff
@@ -1,23 +1,25 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Copyright IBM Corp. - Confidential Information
-from setuptools import setup, find_packages
-
-PUBLISH_VERSION = "1.0.0"
 
+from setuptools import setup, find_packages
 
 setup(
     name='fn_ioc_parser',
-    version=PUBLISH_VERSION,
-    url='https://github.com/ibmresilient/resilient-community-apps',
+    version='1.0.0',
```

Comment:
won't this change now be 1.0.1?

Reply:
@MarkScherfling the function name has changed too so should this version be `v2.0.0` as it will include a breaking change?